### PR TITLE
feat(normalize-docs): normalize custom elements documentation

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: surf-1492

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit 10f833285cf4e5a7e2303c5e662458b7612a73fd

https://app.netlify.com/sites/helix-ui/deploys/5bfc24f440e2a471566ffb7c

File: docs/elements/hx-drop-zone/index.html [hunk 1]
```diff
@@ -42,25 +42,29 @@
 {% endblock %}
 
 {% block attributes %}
-  <dt>drag {String}</dt>
+  <dt>
+    drag {String}
+    <hx-pill class="hxSubBody">read-only</hx-pill>
+  </dt>
     <dd>
```
Comment (CITguy) [inline docs/elements/hx-drop-zone/index.html]: Please wrap in `<dl>` and fix indentation.

File: docs/elements/hx-drop-zone/index.html [hunk 2]
```diff
@@ -42,25 +42,29 @@
...
-      <i>(read-only)</i>
       <p>
         Its value is automatically set by the element itself and mainly used for CSS styling purposes.
       </p>
     </dd>
 {% endblock %}
 
 {% block properties %}
-  <dt>drag {String}</dt>
```
Comment (CITguy) [inline docs/elements/hx-drop-zone/index.html]: `<dl>` like above

File: docs/elements/hx-menu/index.html [hunk 3]
```diff
@@ -60,42 +60,46 @@
 {% endblock %}
 
 {% block properties %}
-  <dl>
-    <dt>controlElement (HTMLElement)</dt>
+  <dl class="memberList">
+    <dt>
+      controlElement (HTMLElement)
```
Comment (CITguy) [inline docs/elements/hx-menu/index.html]: Should be `{HTMLElement}`

File: docs/elements/hx-menu/index.html [hunk 4]
```diff
@@ -60,42 +60,46 @@
...
+    <dt>position {String ["bottom-start"]}</dt>
     <dd>
       <p>
         Manipulates the <code>position</code> attribute.
       </p>
     </dd>
 
-    <dt>relativeElement (HTMLElement)</dt>
+    <dt>
+      relativeElement (HTMLElement)
```
Comment (CITguy) [inline docs/elements/hx-menu/index.html]: `{HTMLElement}`

File: docs/elements/hx-search-assistance/index.html [hunk 5]
```diff
@@ -48,41 +48,45 @@
...
     <dd>ID of an element that the search assistance is positioned relative to.</dd>
   </dl>
 {% endblock %}
 
 {% block properties %}
-  <dl>
-    <dt>controlElement (HTMLElement)</dt>
+  <dl class="memberList">
+    <dt>
+      controlElement (HTMLElement)
```
Comment (CITguy) [inline docs/elements/hx-search-assistance/index.html]: `{HTMLElement}`

File: docs/elements/hx-search-assistance/index.html [hunk 6]
```diff
@@ -48,41 +48,45 @@
...
+    <dt>position {String [bottom-start]}</dt>
     <dd>
       <p>
         Manipulates the <code>position</code> attribute.
       </p>
     </dd>
 
-    <dt>relativeElement (HTMLElement)</dt>
+    <dt>
+      relativeElement (HTMLElement)
```
Comment (CITguy) [inline docs/elements/hx-search-assistance/index.html]: `{HTMLElement}`

File: docs/elements/hx-search-assistance/index.html [hunk 7]
```diff
@@ -48,41 +48,45 @@
     <dt>position {String="bottom-start"}</dt>
```
Comment (CITguy) [inline docs/elements/hx-search-assistance/index.html]: `{String [bottom-start]}`

File: docs/elements/hx-toast/index.html [hunk 8]
```diff
@@ -52,13 +52,13 @@ <h2 id="methods">Methods</h2>
...
   <dl>
-    <dt>cta {String} (optional)</dt>
+    <dt>cta {String} <i>(optional)</i></dt>
     <dd>
       Set the text of the CTA button.
       If absent or blank, the CTA button will not be displayed.
     </dd>
 
-    <dt>type {Enum&lt;String&gt;} (optional)</dt>
+    <dt>type {Enum&lt;String&gt;} <i>(optional)</i></dt>
```
[the remark this answers is not in the document]
Comment (CITguy) [reply docs/elements/hx-toast/index.html]: Let's hold off on standardizing this for now. Attributes may be a special case for how they need to be documented.

**Pull request merged**